src/testRunner/unittests/services/findAllReferences.ts

```diff
@@ -0,0 +1,140 @@
+import { protocol } from "../../_namespaces/ts.server";
+import { createSession } from "../tsserver/helpers";
+import { createServerHost, File } from "../virtualFileSystemWithWatch";
+
+describe("unittests:: services:: findAllReferences", () => {
+    it("does not try to open a file in a project that was updated and no longer has the file", () => {
+        const files: File[] = [
+            {
+                path: "/packages/babel-loader/tsconfig.json",
+                content:
+`
+{
+    "compilerOptions": {
+        "target": "ES2018",
+        "module": "commonjs",
+        "strict": true,
+        "esModuleInterop": true,
+        "composite": true,
+        "rootDir": "src",
+        "outDir": "dist"
+    },
+    "include": ["src"],
+    "references": [{"path": "../core"}]
+}
+`
+            },
+            {
+                path: "/packages/babel-loader/src/index.ts",
+                content:
+`
+import type { Foo } from "../../core/src/index.js";
+`
+            },
+            {
+                path: "/packages/core/tsconfig.json",
+                content:
+`
+{
+    "compilerOptions": {
+        "target": "ES2018",
+        "module": "commonjs",
+        "strict": true,
+        "esModuleInterop": true,
+        "composite": true,
+        "rootDir": "./src",
+        "outDir": "./dist",
+    },
+    "include": ["./src"]
+}
+`
+            },
+            {
+                path: "/packages/core/src/index.ts",
+                content:
+`
+import { Bar } from "./loading-indicator.js";
+export type Foo = {};
+const bar: Bar = {
+    prop: 0
+}
+`
+            },
+            {
+                path: "/packages/core/src/loading-indicator.ts",
+                content:
+`
+export interface Bar {
+    prop: number;
+}
+const bar: Bar = {
+    prop: 1
+}
+`
+            },
+        ];
+        const host = createServerHost(files);
+        const session = createSession(host);

```

Comment:
Can you instead baseline the tsserver log. 
eg of baseline: https://github.com/microsoft/TypeScript/blob/main/src/testRunner/unittests/tsserver/configuredProjects.ts#L55